docs/derivation-rules.saty

```diff
@@ -153,6 +153,36 @@ but extended by remote call, local call, list, etc.
         }
       );
     >
+    +small-block<
```

Comment:
Could you add `[e | e] | []` into BNF?

Reply:
OK. I'll add it.

Reply:
I added it in https://github.com/dwango/fialyzer/pull/191/commits/aa4740d501a0f73ac3e2ef6b0fed7771f0180f6c